examples/digital_out/update_lines.py

```diff
@@ -0,0 +1,20 @@
+"""Example for generating digital signal.
+
+This example demonstrates how to write values to a digital
+output channel.
+"""
+
+import nidaqmx
+from nidaqmx.constants import LineGrouping
+
+with nidaqmx.Task() as task:
+    data = [1, 2, 4, 8]
+
+    task.do_channels.add_do_chan(
+        "Dev1/port0/line0:3", line_grouping=LineGrouping.CHAN_FOR_ALL_LINES
```

Comment:
I would expect update_lines.py to use CHAN_PER_LINE like you did for DI read_lines.py